docs/_sources/user/alff.rst

```diff
@@ -44,6 +44,17 @@ Configuring CPAC to Run ALFF and f/ALFF
    :start-at: amplitude_low_frequency_fluctuation:
    :end-before: regional_homogeneity:
 
+Surface-based f/ALFF
+^^^^^^^^^^^^^^^^^^^^
+.. versionadded:: 1.8.7
+
+C-PAC now offers the computation of surface-based ALFF and fALFF. This can be configured in under surface_analysis in the pipeline file. 
```

Comment:
```suggestion
C-PAC now offers the computation of surface-based ALFF and fALFF. This can be configured under surface_analysis in the pipeline file. 
```